docs/contributing/developing.rst

```diff
@@ -116,6 +116,35 @@ If your Elasticsearch instance is located somewhere else, you can set the
 
     $ ELASTICSEARCH_URL=http://my-elasticsearch-instance:9200 python runtests.py --elasticsearch
 
+**Browser and device support**
+
+Wagtail is meant to be used on a wide variety of devices and browsers. Supported browser / device versions include:
+
+=============  =============  =============
+Browser        Device/OS      Version(s)
+=============  =============  =============
+Mobile Safari  iOS Phone      Latest
+Mobile Safari  iOS Tablet     Latest
+Chrome         Android        Latest
+IE             Desktop        9, 10, 11
```

Comment:
As far as official support goes, Internet Explorer 11 is the last supported version by Microsoft. 
So i guess it is safe to say we can deprecate support for IE 9 and 10. See also: https://www.microsoft.com/en-us/windowsforbusiness/end-of-ie-support

For the past year we have not included it in our Definition of Done for a variety of clients and there were no discussions regarding it (including corporates).